extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/handler/HomematicThingHandler.java

```diff
@@ -456,15 +456,15 @@ private HomematicGateway getHomematicGateway() throws GatewayNotAvailableExcepti
     @Override
     public void handleConfigurationUpdate(Map<String, Object> configurationParameters)
             throws ConfigValidationException {
-        validateConfigurationParameters(configurationParameters);
+        super.handleConfigurationUpdate(configurationParameters);
```

Comment:
I do not see that `HomematicThingHandler` is overriding the `handleConfigurationUpdate` method. So why do you need to add `super` here?

Reply:
This code is inside the overridden handleConfigurationUpdate method... did you miss that? If not, I am confused.

Reply:
@FStolte Sorry I was confused as well :P I meant the `BaseThingHandler` does not have a `handleConfigurationUpdate`, but of course it does... Was late yesterday and woarm in the office, so forget my comment. sorry.